test/manipulation.test.js

```diff
@@ -659,10 +659,60 @@ describe('manipulation', function() {
   });
 
   describe('updateOrCreate', function() {
+    var db = getSchema();
+    var Todo;
+
+    before(function setUp(done) {
+      Todo = db.define('Todo', {
+        content: String,
+      });
+      db.automigrate('Todo', done);
+    });
+    afterEach(function tearDown(done) {
+      db.automigrate(done);
+    });
+
     it('has an alias "patchOrCreate"', function() {
       StubUser.updateOrCreate.should.equal(StubUser.patchOrCreate);
     });
 
+    it('creates a model when one does not exist', function(done) {
+      Todo.updateOrCreate({ content: 'a' }, function(err, data) {
+        if (err) return done(err);
+
+        should.exist(data);
+        should.exist(data.content);
+        data.content.should.equal('a');
```

Comment:
This checks only what has been returned by `updateOrCreate`. A safer version is to load the model instance via `findById` and check what has been stored in the database.